dll/win32/shell32/CDefView.cpp

```diff
@@ -3265,11 +3272,16 @@ HRESULT CDefView::_MergeToolbar()
 
 HRESULT CDefView::_DoFolderViewCB(UINT uMsg, WPARAM wParam, LPARAM lParam)
 {
-    if (m_pShellFolderViewCB)
+    if (!m_pShellFolderViewCB)
     {
-        return m_pShellFolderViewCB->MessageSFVCB(uMsg, wParam, lParam);
+        m_pShellFolderViewCB = _GetDefViewCB(this);
+        m_pShellFolderViewCB->AddRef();
```

Comment:
Why is there an extra `AddRef` here?
Assigning to a smart pointer automatically adds a reference.